kernel/arch/dreamcast/include/dc/pvr.h

```diff
@@ -2270,6 +2271,43 @@ void pvr_poly_cxt_txr_mod(pvr_poly_cxt_t *dst, pvr_list_t list,
     Helper functions for handling texture tasks of various kinds.
 */
 
+/** \brief   Set the global stride width for non-power-of-two textures in PVR RAM.
+    \ingroup pvr_txr_mgmt 
+
+    This function configures the global texture stride register 
```

Comment:
You need to also mention that this stride is not just for any sizes but is only supported for multiples of 32 here.